<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>policy_config</class>
 <widget class="QDialog" name="policy_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="day_of_week_form">
     <item>
      <widget class="QCheckBox" name="day_of_week_0">
       <property name="text">
        <string>Sun</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="day_of_week_1">
       <property name="text">
        <string>Mon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="day_of_week_2">
       <property name="text">
        <string>Tue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="day_of_week_3">
       <property name="text">
        <string>Wed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="day_of_week_4">
       <property name="text">
        <string>Thu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="day_of_week_5">
       <property name="text">
        <string>Fri</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="day_of_week_6">
       <property name="text">
        <string>Sat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="start_time_label">
           <property name="text">
            <string>Start Time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="start_time">
           <property name="time">
            <time>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="end_time_label">
           <property name="text">
            <string>End Time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="end_time">
           <property name="time">
            <time>
             <hour>12</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>gilgil</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IPhone-b</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gilgil-notebook</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DESKTOP-e</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
